### DISCUSSION TITLE
v0.4.0-alpha

### DISCUSSION THREAD

**Discussion by bowring** [CIRDLES/Tripoli#193] [Show and tell]
## What's Changed
* Added plot of intensity residuals and reader for TIMSDP data. by @bowring in https://github.com/CIRDLES/Tripoli/pull/192

**Full Changelog**: https://github.com/CIRDLES/Tripoli/compare/v0.3.9-alpha...v0.4.0-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/CIRDLES/Tripoli/releases/tag/v0.4.0-alpha'>v0.4.0-alpha</a>.</em>

**Comment by noahmclean** [comment]
These residual plots look fantastic!  Lots of insight available here.  I like the big plot with checkboxes for displaying each species together or separately, just like the OnPeak tab.  I'll put some suggestions into Issues for improving navigation and display.  Very cool!
